### PR TITLE
MEN-5508 feat: dockerevents description update.

### PR DESCRIPTION
**Pull request by merlin-northern**
goes with:
* https://github.com/mendersoftware/monitor-client/pull/120
* https://github.com/mendersoftware/integration/pull/1989

Changelog: title
Ticket:MEN-5508
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md [hunk 1]
```diff
@@ -128,6 +128,43 @@ With the above you can configure alerting based on any source.
...
+
+For instance, to monitor for `kill` event on a container named `scanner` you need to create a check with `mender-monitorctl`
+command and enable it in the following way:
+
+```bash
+sudo mender-monitorctl create dockerevents scanner_kill scanner kill 16
+sudo mender-monitorctl enable dockerevents scanner_kill
+```
+
+Where the last and optional argument to `mender-monitorctl create` stands for a number of seconds after which
```
Comment (oleorhagen) [inline 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md:146]: Nit: `the` number of seconds.

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md [hunk 2]
```diff
@@ -128,6 +128,43 @@ With the above you can configure alerting based on any source.
...
+the Mender Monitor daemon sends an automatic OK.
```
Comment (oleorhagen) [inline 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md:147]: Suggestion: Expand on this a little bit.

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md [hunk 3]
```diff
@@ -128,6 +128,43 @@ With the above you can configure alerting based on any source.
...
+> LOG_PATTERN=".*container kill.*name=scanner.*"
+> LOG_FILE="@docker events"
+> LOG_PATTERN_EXPIRATION=16
+> LOG_ALERT_DESCRIPTION="Docker container scanner kill"
+> LOG_ALERT_DETAILS="Alert was raised due to:%line_matching"
+> LOG_ALERT_STATUS=DOCKEREVENTS_CONTAINER_RESTART
+> LOG_ALERT_TYPE=docker_event
+> ```
+
+You can also consult `mender-monitorctl help` for a summary of options and online help.
```
Comment (oleorhagen) [inline 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md:166]: Nice, but maybe this should be somewhere else? 

Or maybe in every section? :thinking: 

Not really sure

**Review by oleorhagen — APPROVED** [review]
Nice, left a few minor discussion point only!

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md (hunk 3 above)
Comment (merlin-northern) [reply 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md]: moved to [the top](https://github.com/mendersoftware/mender-docs/compare/44354dbef438cfb55b9137acaec2aebfbc3cbe09..f3e54f28dc657f9b208db92744732db1bba1eafa#diff-a67e137dae3b3404bf7eb54ceb4db082e56c3d15d838f2fe56b91d192cdf6e80R12)

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md (hunk 2 above)
Comment (merlin-northern) [reply 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md]: expanded.

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md (hunk 1 above)
Comment (merlin-northern) [reply 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md]: fixed.

File: 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md (hunk 3 above)
Comment (oleorhagen) [reply 09.Add-ons/20.Monitor/20.Monitoring-subsystems/docs.md]: Total sense makin'

**Pull request merged**